tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is this file getting used?

Reply:
That was not supposed to be added. I decided to do this in a different PR. Removed.